test/test_util.py

```diff
@@ -1,18 +1,15 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
-import io
-import locale
+import locale # noqa F401 imported but unused
```

Comment:
The `locale` library is not used and in the code of the file it is not found anywhere else, but when I remove it I get an error in [AppVeyor CI]( https://ci.appveyor.com/project/airspeed-velocity/asv/builds/42784236). Any idea why this happens?

Reply:
Can you move this to it's own PR, so we can isolate and see the CI failure without other changes.